src/styles/blocks/card.scss

```
@@ -0,0 +1,93 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  background: #fff;
+  border: 1px solid #e5e5e5;
+  border-radius: 5px;
+  max-width: $card-width;
```

```suggestion
  width: $card-width;
```
It's just width, can't be less